zh_CN/messages.json

```diff
@@ -846,7 +852,7 @@
         "message": "上一行"
     },
     "Line_break": {
```

Comment:
@derjanb Can you give me the context of this string? This way I can check whether my own translation is correct.

Reply:
It is an editor option as to whether long lines should be automatically wrapped.

Reply:
thanks